### DISCUSSION TITLE
Any way to change Botany Pots times in config?

### DISCUSSION THREAD

**Discussion by Redeix** [NillerMedDild/Enigmatica6#3527] [Q&A] [answered]
looked around in config but none of my changes affected the times in game.

**Comment by MuteTiefling** [comment, marked as answer]
https://github.com/NillerMedDild/Enigmatica6/blob/6f4774bf7bce46208971e965664cc17a8bf9e83b/kubejs/server_scripts/enigmatica/kubejs/base/unification/unify_growables.js#L58

**Reply by Redeix** [reply]
Thanks a bunch!

**Comment by MuteTiefling** [comment]
Just be careful. We slowed them down to help with lag. And do be aware that the current output per minute is -more- than default. It's just coming in bigger chunks less frequently.
